app/src/components/inputs/inputUserSearch/inputUserSearch.jsx

```diff
@@ -61,51 +65,81 @@ const makeURL = (input, isAdmin, projectId) => {
   startUrl += `?${stringify(qsParams, { encode: false })}`;
   return startUrl;
 };
-const makeOptions = (options, projectId) => options.map(option => ({
-  userName: option.full_name || '',
-  userLogin: option.userId,
-  email: option.email || '',
-  disabled: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
-  isAssigned: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
-  userAvatar: getPhoto(option.userId),
-}));
+const makeOptions = (options, projectId) =>
+  options.map((option) => ({
+    userName: option.full_name || '',
+    userLogin: option.userId,
+    email: option.email || '',
+    disabled: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
+    isAssigned: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
+    userAvatar: getPhoto(option.userId),
+  }));
 const getUsers = (input, isAdmin, projectId) => {
   if (input) {
     const url = makeURL(input, isAdmin, projectId);
-    return fetch(url)
-      .then((response) => {
-        const arr = makeOptions(response.content, projectId);
-        return ({ options: arr });
-      });
+    return fetch(url).then((response) => {
+      const arr = makeOptions(response.content, projectId);
+      return { options: arr };
+    });
   }
   return Promise.resolve({ options: [] });
 };
 
-export const InputUserSearch = ({ isAdmin, onChange, projectId }) => (
-  <Select.AsyncCreatable
-    cache={false}
-    className={cx('select2-search-users')}
-    loadOptions={input => (getUsers(input, isAdmin, projectId))}
-    filterOption={() => (true)}
-    loadingPlaceholder={<FormattedMessage id={'InputUserSearch.searching'} defaultMessage={'Searching...'} />}
-    noResultsText={<FormattedMessage id={'InputUserSearch.noResults'} defaultMessage={'No matches found'} />}
-    searchPromptText={<FormattedMessage id={'InputUserSearch.placeholder'} defaultMessage={'Please enter 1 or more character'} />}
-    onChange={(option) => { onChange(option); }}
-    menuRenderer={({ focusOption, options, selectValue }) =>
-      (UsersList({ focusOption, options, selectValue }))}
-    isValidNewOption={isValidNewOption}
-    newOptionCreator={newOptionCreator}
-    promptTextCreator={promptTextCreator}
-  />
+export const InputUserSearch = ({
+  isAdmin,
+  onChange,
+  projectId,
+  placeholder,
+  value,
+  error,
+  touched,
+}) => (
+  <div className={cx('select2-search-users')}>
+    <Select.AsyncCreatable
+      className={cx({ error: error && touched })}
+      value={value}
+      cache={false}
+      loadOptions={(input) => getUsers(input, isAdmin, projectId)}
+      filterOption={() => true}
+      loadingPlaceholder={
+        <FormattedMessage id={'InputUserSearch.searching'} defaultMessage={'Searching...'} />
```

Comment:
For string type arguments of react component, we can use simplified notation (see below). Please ignore it if is a recommendation of a code style guide
`<FormattedMessage id="InputUserSearch.searching"defaultMessage="Searching..." />`